test/e2e/redux/reducers/register.reducer.spec.ts

```diff
@@ -96,6 +101,11 @@ describe("register reducer", () => {
     it("should handle s", () => {
         expect(
             reducer({
+                auth: new CGAccount({
+                    name: "",
+                    directory: "",
+                    sendStats: false
+                }),
```

Comment:
Create duplicate of this file called auth.reducer.spec and test your auth actions